### ISSUE TITLE
feat (desktop): Hide the root domain in a toggle block so Apple stops complaining about it

### ISSUE THREAD

**Issue by xavierchanth** [atsign-foundation/noports#1603]
- Apple keeps complaining about how the registrar infinitely loads when they change the root domain.
- This feature is only for people hosting their own registrar or atDirectory so in most cases it should be concealed.
  - It still needs to be accessible exactly where it is, but we should hide it under a toggle block that is called `Advanced Options` or something similar.

**Comment by CurtlyCritchlow** [comment]
@XavierChanth I think we should close this ticket. Since apple approval is no longer an issue.

**Comment by xavierchanth** [comment]
Yes, we can, also @Zambrella's new onboarding UI (I have to complete some more changes to onboarding libraries before it's ready) will hide it in a toggle block.
